scripts/o3de/o3de/engine_template.py

```diff
@@ -2112,7 +2114,8 @@ def create_gem(gem_path: pathlib.Path,
     
     tags = [gem_name]
     if user_tags:
-        new_tags = user_tags.split() if isinstance(user_tags, str) else user_tags
+        # Allow commas or spaces as tag separators
+        new_tags = user_tags.replace(',', ' ').split() if isinstance(user_tags, str) else user_tags
```

Comment:
How does this handle filtering for lines with commas and spaces?

For instance, if I have `tag1, tag2`, does it count only 2 elements, or will it count 3 (where one of them is empty because its surrounded by 2 empty spaces)?

Reply:
![image](https://user-images.githubusercontent.com/26804013/192396126-556c524f-4f18-4fa9-bbdc-cdacd0f2276f.png)

Reply:
Cool, seems straightforward to me then